### ISSUE TITLE
Cache Audio for offline use

### ISSUE THREAD

**Issue by emme10** [danielR2001/flutter_exoplayer#6]
Is it possible to cache remotefiles on the user device to play them offline if the connection with the internet fails. Something like spotify does without downloading the whole file everytime ?

**Comment by danielR2001** [comment]
Yes, Spotify uses the same player like me (ExoPlayer), this feature will be added later to the library.
